CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
+## [5.1.0]
```

Comment:
Other items that I would add based on checking the [changes](https://github.com/halestudio/hale-cli/compare/v4.0.0...devel):

### Added

- Added support for specifying filters when loading data

### Changed

- Always use the latest versions of hale offline resources like INSPIRE schemas